### DISCUSSION TITLE
Sprite Packs Combined issue

### DISCUSSION THREAD

**Discussion by impurity-games** [Monika-After-Story/MonikaModDev#9310] [QA - tech support + general] [answered]
So i tried downloading the sprite packs off the website following the guide as closely as I could and all the files are there but it's not showing up in game. I checked the log and whatnot and everything is listed there too. Is this an error or do i have to reach a certain affection level or something?

**Comment by Iceninja64** [comment, marked as answer]
Have you gifted some of sprite packs because you can download them but if you didn't give them to monika she won't know and I think you need to have some affection to change rooms
